assets/styles/styles.css

```diff
@@ -257,68 +253,68 @@ strong {
     overflow-x: hidden;
   }
 
-._menu {
-  margin-left: 15px;
-}
+  .menu {
+    margin-left: 15px;
```

Comment:
You are expected to use a breakpoint greater than 768px for all your projects that is from mobile to desktop